server/routes/event.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const eventController = require('../controllers/eventController')
+
+const router = express.Router();
+
+// TODO: GET request routes
+
+router.post('/', eventController.createEvent, (req, res) => {
+  return res.status(200).json(res.locals.newEvent);
+});
+
+router.get('/', eventController.getEvent, (req, res) => {
+  return res.status(200).json(res.locals.event);
+})
+
+router.put('/update', eventController.updateEvent, (req, res) => {
```

Comment:
In order to follow REST, update this so PUT requests go to `/event` directly instead of `/event/update`?